upf/upf.c

```diff
@@ -157,6 +157,8 @@ vnet_upf_nat_pool_add_del (u8 * nwi_name, ip4_address_t start_addr,
   upf_nat_pool_t *nat_pool = NULL;
   uword *p;
 
+  name = vec_dup (name);
```

Comment:
There's second `vec_dup (name)` below which needs to be replaced by just `name`, otherwise we've got a leak